src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/Listeners/BlobListenerFactory.cs

```diff
@@ -8,52 +8,51 @@
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Queues;
 using Microsoft.Azure.WebJobs.Host.Queues.Listeners;
-using Microsoft.Azure.WebJobs.Host.Storage;
-using Microsoft.Azure.WebJobs.Host.Storage.Blob;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
 using Microsoft.Azure.WebJobs.Host.Timers;
 using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Queue;
 
 namespace Microsoft.Azure.WebJobs.Host.Blobs.Listeners
 {
     internal class BlobListenerFactory : IListenerFactory
     {
         private const string SingletonBlobListenerScopeId = "WebJobs.Internal.Blobs";
         private readonly IHostIdProvider _hostIdProvider;
-        private readonly IQueueConfiguration _queueConfiguration;
-        private readonly JobHostBlobsConfiguration _blobsConfiguration;
+        private readonly JobHostQueuesOptions _queueOptions;
+        private readonly JobHostBlobsOptions _blobsOptions;
         private readonly IWebJobsExceptionHandler _exceptionHandler;
         private readonly IContextSetter<IBlobWrittenWatcher> _blobWrittenWatcherSetter;
-        private readonly IContextSetter<IMessageEnqueuedWatcher> _messageEnqueuedWatcherSetter;
+        private readonly SharedQueueWatcher _messageEnqueuedWatcherSetter;
```

Comment:
Awesome that we're getting rid of this IContextSetter stuff. _blobWrittenWatcherSetter is still using one though? Would be good to be consistent

Reply:
That should all be gone.

@MikeStall , thought you had done that as part of your changes. If not, let me know and I'll remove the remaining usages.